content/pages/constitution.md

```diff
@@ -0,0 +1,313 @@
+        minimum of two events per term;
+
+    -   To administer the CSters mailing list.
+
+8.  The duties of the Diretor of Publicity are:
+
+    -   To publicize events and other endeavours of the committee
+        through physical and electronic means;
+
+    -   To design all media circulated by the committee including:
+        posters, social media profiles and the official website.
+
+# Procedures #
+
+Any changes to procedures must be proposed with at least one week's
+notice, and ratified at an official meeting of the undergraduate
+committee.
+
+## Policy ##
+
+1.  Code of Conduct\
+    The Code of Conduct outlines expected behaviour for all members of
+    the community at any event affiliated with the undergraduate
+    committee. Any member of the committee may prpose an amendment to
+    the CoC. The committee must ratify any proposed amendments to the
+    Code of Conduct.
+
+2.  Machine Usage Agreement\
+    The Machine Usage Agreement outlines expected conduct when using any
+    machines maintained by the undergraduate committee. The machine
+    usage agreement is maintained by the systems administrator. Any
+    proposed changes must be first ratified by wics-sys, followed by the
+    entire committee.
+
+3.  Office Poliy\
+    Office Policy outlines expected behaviour in the WiCS Undergraduate
+    Committee office, M3 1003. The office policy is maintained by the
+    Office Manager. Any proposed changes must be ratified by the
+    committee.
+
+## Succession Procedure ##
+
+1.  No later than the end of the first week of the fourth month of the
+    current term, a call for committee members should be announced on
+    the website and sent out to all affiliated mailing lists.
+
+2.  New volunteers caan apply to be on the committee for the following
+    term, and the current committee will select applicants for
+    interviews. A new volunteer shall be anyone who has never served on
+    the committee before, or has not served on the committee for 4 or
+    more terms. Interviewers should be chosen by the current committee.
+    If an interviewer has a conflict of interest with the interviewee,
+    an alternate interviewer will be chosen. In any case, if no
+    interviewer is explicitly chosen, the current Chair shall conduct
+    interviews. The interviews are intended to welcome the new member to
+    the committee, give them background on the committee's activities,
+    and help them find a role for the next term. The current committee
+    will ultimately select the comoposition of the committee for next
+    term.
+
+3.  Existing committee members can nominate themselves for positions on
+    the next term's committee given the following conditions:
+
+    -   No member may serve more than three consecutive terms;
+
+    -   No member may serve as Chair for more than one term
+        consecutively;
+
+    -   No member intending to serve as Chair or Secretary may be on
+        co-op;
```

Comment:
In general, I don't think co-op students should serve on the committee. When special cases arise, I think they should be treated as special cases and not general policy. Indeed, I think it might even make sense to make this stronger, i.e. *No member intending to serve as Chair or Secretary may be on co-op, and students enrolled in full-time studies will be given preference over co-op students;* (but that should also apply to new students, so maybe it doesn't make sense to put it in this section)

The issue with co-op students holding committee positions is that a) even if they are on campus, they are expected to be at work 9-5 and we have a number of required responsibilities during business hours and b) they are working 40 hours per week and it's hard to hold an additional 10h/wk on campus commitment on top of that when the vast majority of our activities take place during weekdays. That's asking for 8+2 = ~10 hour work days for the entire school term, and I don't think that's something we should encourage in policy.